src/dictionary.rs

```diff
@@ -59,8 +59,8 @@ impl Dictionary {
     /// If the dictionary did have this key present, the value is updated, and the old value is
     /// returned.
     #[inline]
-    pub fn insert(&mut self, k: String, v: Value) -> Option<Value> {
-        self.map.insert(k, v)
+    pub fn insert(&mut self, k: impl Into<String>, v: impl Into<Value>) -> Option<Value> {
```

Comment:
This isn't backwards compatible I'm afraid. It would be nice to have `Into<Value>` but I don't it warrants a major version bump.